packages/salesforcedx-vscode-apex-debugger/src/index.ts

```diff
@@ -22,16 +22,6 @@ import {
   WORKSPACE_SETTINGS_REQUEST,
   WorkspaceSettings
 } from '@salesforce/salesforcedx-apex-debugger/out/src';
-import {
-  ENV_SFDX_DEFAULTUSERNAME,
-  ENV_SFDX_INSTANCE_URL,
-  SFDX_CONFIG_ISV_DEBUGGER_SID,
-  SFDX_CONFIG_ISV_DEBUGGER_URL
-} from '@salesforce/salesforcedx-utils-vscode/out/src';
```

Comment:
Interesting. If utils was can't be a direct dependency of salesforcedx-vscode-apex-debugger, I would think you can do `from '@salesforce/salesforcedx-apex-debugger/node_modules/@salesforce/salesforcedx-utils-vscode/out/src`. Does that not work?